src/parameterCfg.m

```diff
@@ -155,7 +155,7 @@
 
 % Reflection Coefficient used if material is not present in material
 % library
-para = fieldToNum(para, 'reflectionCoefficient', [], [0.3162; 0.3162]); % [0.3162; 0.3162] for RL 10 dB
+para = fieldToNum(para, 'reflectionCoefficient', [], [10^(-para.reflectionLoss/20);10^(-para.reflectionLoss/20)]); 
```

Comment:
Also, it would be nice
1. to describe why a 2-component vector is needed (what do the two components refer to)
2. check that what you get as input is what you expect (non-negative column vector with 2 elements?)

Reply:
> thanks for the update.
> One additional doubt: if a user wanted to set a custom reflection coefficient, how would he do it?
> I'm assuming something like adding the following line to the param. config. txt file
> `reflectionCoefficient [0.1; 0.2]`
> 
> Have you tested whether you get the expected behavior in this case?
> Does fieldToNum support string representing vectors as input?

Note that reflectionCoefficient is being used to calculate RL for TGay channel. Thus, user needs to set only reflectionLoss which in turn converted to reflectionCoefficient inside the code.  If material is present, software would calculate reflectionLoss using reflectionCoefficient based on the material library. If material is not present, software would use (i) reflectionLoss if mentioned in the paracnf file (ii) default reflectionLoss if not mentioned in paracnf file. 

In general, user needs to set reflectionLoss parameter in paraconf file for both NIST or TGay channel model. Otherwise, it would consider default values if material is missing

Reply:
I have removed reflectionCoefficient from paracng file. In paracnf file, user just need to specify reflectionLoss in both tgay and NIST channel models.